samcli/commands/local/start_lambda/cli.py

```diff
@@ -0,0 +1,79 @@
+"""
+CLI command for "local start-api" command
```

Comment:
it is not start-api

Reply:
gah... I didn't copy start-api at all... ;)